pkg/cache/cache.go

```diff
@@ -148,6 +148,8 @@ func (c *Cache) newClusterQueue(cq *kueue.ClusterQueue) (*clusterQueue, error) {
 		AdmittedUsage:       make(resources.FlavorResourceQuantities),
 		resourceNode:        NewResourceNode(),
 		tasCache:            &c.tasCache,
+
+		workloadsNotAccountedForTAS: sets.New[string](),
```

Comment:
Maybe the keys should be `NamespacedName` to be extra careful here but since we are using string elsewhere it may not be a problem